src/ModuleList/Lesson/useLesson.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect } from 'react';
```

Comment:
I'm not actually sure that we need this hook. There isn't too much logic in there that I think warrants another hook other than `useLessonMeta`. See my comments on the `Lesson.tsx` file for more info. What are your thoughts on this?

Reply:
I was wondering the same thing, but wasn't sure if I was missing something hehe #codemonkey.
I tried deleting and it looks great.